region/brinstar/pink/Spore Spawn Farming Room.json

```
@@ -369,6 +369,58 @@
         "leaveWithGModeSetup": {}
       },
       "flashSuitChecked": true
+    },
+    	{
+      "link": [2, 2],
+      "name": "Gain Flash Suit (Slopespark)",
+      "requires": [
+        {"or": [
+          {"canShineCharge": {"usedTiles": 20, "steepDownTiles": 2, "openEnd": 0}},
+          {"and": [
+            {"doorUnlockedAtNode": 2},
+            {"canShineCharge": {"usedTiles": 21, "steepDownTiles": 2, "openEnd": 0}}
+          ]}
+        ]},
+        {"shineChargeFrames": 143},
```

```suggestion
        {"shineChargeFrames": 150},
```
We usually round up to multiples of 5 for frame counts. And it's better if the value in logic isn't tighter than what the video demonstrates. For a strat like this where the shinecharge is gained and used in the same room, it's not necessary to include a `shineChargeFrames` requirement at all. Personally I probably wouldn't bother with it except in a case where it was particularly tight (10 or fewer frames remaining). But it's also ok here as a way of describing how much extra time you have.